pyvista/examples/planets.py

```diff
@@ -0,0 +1,744 @@
+    >>> import pyvista
+    >>> from pyvista import examples
+    >>> plotter = pyvista.Plotter()
+    >>> image_path = examples.planets.download_stars_jpg(load=False)
+    >>> plotter.add_background_image(image_path)
+    >>> _ = plotter.add_mesh(examples.planets.load_mars())
+    >>> plotter.show()
+
+    """
+    sphere = _sphere_with_texture_map(
+        radius=radius, lat_resolution=lat_resolution, lon_resolution=lon_resolution
+    )
+    surface = examples.planets.download_mars_jpg()
+    sphere.textures["surface"] = surface
+    return sphere
+
+
+def load_jupiter(radius=1.0, lat_resolution=50, lon_resolution=100):
+    """Load a Jupiter source.
+
+    Parameters
+    ----------
+    radius : float, optional
+        Sphere radius.
+
+    lat_resolution : int, optional
+        Set the number of points in the latitude direction.
+
+    lon_resolution : int, optional
+        Set the number of points in the longitude direction.
+
+    Returns
+    -------
+    pyvista.PolyData
+        Jupiter dataset with texture.
+
+    Examples
+    --------
+    >>> import pyvista
+    >>> from pyvista import examples
+    >>> plotter = pyvista.Plotter()
+    >>> image_path = examples.planets.download_stars_jpg(load=False)
+    >>> plotter.add_background_image(image_path)
+    >>> _ = plotter.add_mesh(examples.planets.load_jupiter())
+    >>> plotter.show()
+
+    """
+    sphere = _sphere_with_texture_map(
+        radius=radius, lat_resolution=lat_resolution, lon_resolution=lon_resolution
+    )
+    atmosphere = download_jupiter_jpg()
+    sphere.textures["atmosphere"] = atmosphere
+    return sphere
+
+
+def load_saturn(radius=1.0, lat_resolution=50, lon_resolution=100):
+    """Load a Saturn source.
+
+    Parameters
+    ----------
+    radius : float, optional
+        Sphere radius.
+
+    lat_resolution : int, optional
+        Set the number of points in the latitude direction.
+
+    lon_resolution : int, optional
+        Set the number of points in the longitude direction.
+
+    Returns
+    -------
+    pyvista.PolyData
+        Saturn dataset with texture.
+
+    Examples
+    --------
+    >>> import pyvista
+    >>> from pyvista import examples
+    >>> plotter = pyvista.Plotter()
+    >>> image_path = examples.planets.download_stars_jpg(load=False)
+    >>> plotter.add_background_image(image_path)
+    >>> _ = plotter.add_mesh(examples.planets.load_saturn())
+    >>> plotter.show()
+
+    """
+    sphere = _sphere_with_texture_map(
+        radius=radius, lat_resolution=lat_resolution, lon_resolution=lon_resolution
+    )
+    atmosphere = download_saturn_jpg()
+    sphere.textures["atmosphere"] = atmosphere
+    return sphere
+
+
+def load_saturn_ring_alpha(*args, **kwargs):
+    """Load a source for Saturn's rings with opacity.
+
+    Arguments are passed on to :func:`pyvista.Disc`.
+
+    Parameters
+    ----------
+    *args
+        Variable length argument list.
+
+    **kwargs
+        Arbitrary keyword arguments.
```

Comment:
Shouldn't we document the options properly? Or instead of passing everything on to `pv.Disc`, define a set of meaningful names, just like `lat_resolution`?

Reply:
```suggestion
    Parameters
    ----------
    inner : float, optional
        The inner radius.

    outer : float, optional
        The outer radius.

    c_res : int, optional
        The number of points in the circumferential direction.
```